matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -88,13 +92,90 @@ pub struct WebRtcSocketConfig {
     /// or: `wss://matchbox.example.com/your_game?next=2`
     ///
     /// The last form will pair player in the order they connect.
-    pub room_url: String,
+    pub(crate) room_url: String,
     /// Configuration for the (single) ICE server
-    pub ice_server: RtcIceServerConfig,
+    pub(crate) ice_server: RtcIceServerConfig,
     /// Configuration for one or multiple reliable or unreliable data channels
-    pub channels: Vec<ChannelConfig>,
+    pub(crate) channels: Vec<ChannelConfig>,
     /// The amount of attempts to initiate connection
-    pub attempts: Option<u16>,
+    pub(crate) attempts: Option<u16>,
+}
+
+impl WebRtcSocketBuilder {
+    /// Creates a new builder for a connection to a given room with a given number of
+    /// re-connection attempts.
+    ///
+    /// You must add at least one channel with [`WebRtcSocketBuilder::add_channel`],
+    /// [`WebRtcSocketBuilder::add_reliable_channel`], or
+    /// [`WebRtcSocketBuilder::add_unreliable_channel`] before you can build the
+    /// [`WebRtcSocket`]
+    pub fn new(room_url: impl ToString, attempts: Option<u16>) -> Self {
+        Self {
+            room_url: room_url.to_string(),
+            ice_server: RtcIceServerConfig::default(),
```

Comment:
It's no longer possible to set your own ice server config?

Reply:
Oops :rofl: Will fix that